src/overview.md

```diff
@@ -183,27 +183,68 @@ To find additional APIs, see the
 [commonly used packages page](/guides/libraries/useful-libraries).
 
 
-## Dart: The platform
+## Dart: The platform {#platform}
 
 Dart's compiler technology lets you run code in different ways:
 
-* **Native compilation**: For apps targeting mobile and desktop devices,
+* **Native platform**: For apps targeting mobile and desktop devices,
   Dart includes both a Dart VM with just-in-time (JIT) compilation and
   an ahead-of-time (AOT) compiler for producing machine code.
 
-* **Web compilation**: For apps targeting the web,
+* **Web platform**: For apps targeting the web,
   Dart includes both a development time compiler (dartdevc) and
   a production time compiler (dart2js).
   Both compilers translate Dart into JavaScript.
 
 <img src="{% asset Dart-platforms.svg @path %}" width="800px" alt="An
 illustration of the targets supported by Dart">
 
-In addition, [Flutter]({{site.flutter}}) provides tooling and UI libraries to
-build UIs that run on iOS, Android, macOS, Windows, and Linux.
+The [Flutter framework]({{site.flutter}}) is a popular multi-platform UI toolkit
+that's powered by the Dart platform, and which provides tooling and UI libraries
+to build UI experiences that run on iOS, Android, macOS, Windows, Linux, and the
+Web.
 
+#### Dart Native (machine code JIT and AOT) {#native-platform}
 
-## Learning Dart
+During development, aa fast developer cycle is critical for iteration. The Dart
+VM offers a just-in-time compiler (JIT) that supports both pure interpretation
+(as required on iOS devices, for example) and runtime optimization.
+
+When apps are ready to be deployed to production — whether you're publishing to
+an app store or deploying to a production backend — the Dart AOT compiler
+enables ahead-of-time compilation to native ARM or X64 code machine code. Your
+AOT-compiled app starts instantly and runs smoothly.
```

Comment:
is "instantly" a little provocative? And it running smoothly is a factor of the app, rather than merely the compiler.